src/Mappers/CannotMapTypeException.php

```diff
@@ -177,4 +180,19 @@ public static function createForNonNullReturnByTypeMapper(): self
     {
         return new self('a type mapper returned a GraphQL\Type\Definition\NonNull instance. All instances returned by type mappers should be nullable. It is the role of the NullableTypeMapperAdapter class to make a GraphQL type in a "NonNull". Note: this is an error in the TypeMapper code or in GraphQLite itself. Please check your custom type mappers or open an issue on GitHub if you don\'t have any custom type mapper.');
     }
+
+    public static function createForUnexpectedCallableParameters(): self
```

Comment:
Not a fan of having these (type mapper specific) constructors here, but that's the way it's done for every other mapper 🤔